test/_common.py

```diff
@@ -58,3 +61,28 @@ def null_logger():
     logger = logging.getLogger("null")
     logger.setLevel(logging.ERROR)
     return logger
+
+class MockResponse():
+    """
+    A simple object which contains all the fields we need from a response
+
+    :param str text: the text to be returned with the response
+    """
+    def __init__(self, text):
+        self.headers = {"content-type": "text/xml"}
+        self.status_code = 200
+        self.text = text
+
+def simulate_bgg(url, params, timeout):
+    last_slash = url.rindex('/')
+    fragment = url[last_slash + 1:]
+
+    sorted_params = sorted(params.items(), key=lambda t: t[0])
+    query_string = '&'.join([str(k) + "=" + str(v) for k, v in sorted_params])
+
+    filename = os.path.join(XML_PATH, fragment + "?" + query_string)
+
+    with open(filename, "r") as xmlfile:
```

Comment:
I had to decode utf-8 here to get the tests to work on Python 2.7.

`with io.open(filename, "r", encoding="utf-8") as xmlfile:`